src/libs/Clipboard/index.native.js

```diff
@@ -1,3 +1,9 @@
 import Clipboard from '@react-native-community/clipboard';
 
-export default Clipboard;
+export default {
+    ...Clipboard,
+
+    // Native does not support HTML.
```

Comment:
```suggestion
    // We don't want to set HTML on native platforms so noop them. 
```
In fact, HTML copying and pasting is supported natively.

Reply:
In RN?

Reply:
Yeah, you can do this via native modules. This lib that we are using doesn't support it yet.